sensor/Dockerfile.raspberrypi4-64

```diff
@@ -0,0 +1,25 @@
+FROM balenalib/raspberrypi3-python:3-build
+
+ARG BSEC_FILENAME=BSEC_1.4.7.4_Generic_Release.zip
+
+RUN install_packages \
+        unzip
+
+WORKDIR /usr/src/app
+RUN git clone https://github.com/balena-io-playground/bsec_bme680_linux.git
+RUN wget https://ae-bst.resource.bosch.com/media/_tech/media/bsec/$BSEC_FILENAME
+RUN unzip -d bsec_bme680_linux/src $BSEC_FILENAME
+RUN echo $BSEC_FILENAME > version
+
+WORKDIR /usr/src/app/bsec_bme680_linux
+RUN chmod +x make.sh
+RUN ./make.sh
+
+RUN pip install smbus
+
+WORKDIR /usr/src/app
+COPY ./scripts ./scripts
+COPY ./entry.sh /usr/src/app/
+RUN chmod +x /usr/src/app/entry.sh
+
+CMD ./entry.sh
```

Comment:
if doing `/bin/bash ./entry.sh` for example, or `/bin/bash ./entry.sh`, then no need for the chmod step. Those are out of place for me, but maybe it's just a matter of liking.
(thus this is a `nit` :)

Reply:
I really only started adding the chmod step workaround to [this cli issue](https://github.com/balena-io/balena-cli/issues/1323), perhaps your suggestion is tidier!